### DISCUSSION TITLE
Facing issue while uploading the document for student developer pack

### DISCUSSION THREAD

**Discussion by ns-0437** [community/community#65201] [GitHub Education]
Having an issue while applying for the student developer pack

General

### Body

After uploading my institute ID card I am facing an issue with document verification, they are not accepting my document.
I have applied for the student developer pack four times every time I am getting the same reason for rejection although I am uploading the correct documents.
I am also attaching the file which I have uploaded to the site although my institute name is mentioned there, I am getting this error.
The error I am getting is mentioned below
"Our application cannot be reviewed until you fix the following:
The image you selected does not appear to contain your school name. Ensure your image (photo or upload) is legible. We do not accept school logos. If you are taking a photo of a screen try to have it as large a resolution as possible. You may include multiple documents in your image. If your original document is not in English then you might take a photo of the original next to an English translation."
![Screenshot 2023-08-28 194757](https://github.com/community/community/assets/98084760/f0aa0113-11b2-4326-b9cb-2acdc8c3ba26)

**Comment by dhanush1092003** [comment]
Upload different academic proof and try again like enrollment letter or fee recipts

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @ns-0437 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. 


Additionally, I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).
